script/linux

```diff
@@ -3,10 +3,17 @@
 # if not on Linux, do nothing
 [[ $(uname) == "Linux" ]] || exit 0
 
-# Copy settings and keymap to the user's home directory if they don't exist
+# Copy assets to the user's home directory if they don't exist
 mkdir -p "$HOME/.config/zed"
+
+mkdir -p "$HOME/.config/zed/plugins"
+
+mkdir -p "$HOME/.config/zed/themes"
+cp -ruL ./assets/themes/*/*.json "$HOME/.config/zed/"
```

Comment:
@aminya This copies into ~/.config/zed, not into ~/.config/zed/themes.

Reply:
Good catch. Fixed in [`73fe59b` (#2)](https://github.com/kvark/zed/pull/2/commits/73fe59b692295e8ee3b517e05c79b28b8261b566)